apps/molgenis-components/src/components/tables/RoutedTableExplorer.vue

```diff
@@ -0,0 +1,223 @@
+<template>
+  <div>
+    <TableExplorer
+      :tableName="tableName"
+      :graphqlURL="graphqlURL"
+      :canEdit="canEdit"
+      :canManage="canManage"
+      @update:conditions="updateConditions"
+      @update:showColumns="updateColumns"
+      @update:showFilters="updateFilters"
+      @update:showPage="updatePage"
+      @update:showLimit="updateLimit"
+      @update:showOrder="updateOrder"
+      @update:showView="updateView"
+      :showView="getView()"
+      :showColumns="getColumns()"
+      :showFilters="getFilters()"
+      :conditions="getConditions()"
+      :showPage="getPage()"
+      :showLimit="getLimit()"
+      :showOrderBy="getOrderBy()"
+      :showOrder="getOrder()"
+    />
+  </div>
+</template>
+
+<script>
+import TableExplorer from "./TableExplorer.vue";
+
+export default {
+  name: "RoutedTableExplorer",
+  components: {
+    TableExplorer,
+  },
+  data() {
+    return {
+      allColumns: [],
+    };
+  },
+  props: {
+    tableName: {
+      type: String,
+      required: true,
+    },
+    graphqlURL: {
+      type: String,
+      default: () => "graphql",
+    },
+    canEdit: {
+      type: Boolean,
+      default: () => false,
+    },
+    canManage: {
+      type: Boolean,
+      default: () => false,
+    },
+  },
+  methods: {
+    getOrderBy() {
```

Comment:
why are these al methods ( instead of computed values) do you explicitly want to stop reactivity here ?

Reply:
We only want in for initialization, so we don't want to have any reactivity.